app/ui/src/app/integration/edit-page/step-configure/step-configure.component.ts

```diff
@@ -33,6 +33,7 @@ export class IntegrationStepConfigureComponent implements OnInit, OnDestroy {
   routeSubscription: Subscription;
 
   constructor(
+    private cd: ChangeDetectorRef,
```

Comment:
Unless we eventually get away with bloating the application with esoteric class members and variables such as `dc`, `x`, `res` and the like, let's stick to the principle of naming variables and class properties with names matching their type.

Unfortunately, this application does not honor the Single Responsibility Principle, so it's not rare to find class structures with literally 600 LOC or even more. Creating names that self-describe the types in use will spare us from having to inspect entities to infer their type and purpose hence gaining a better understanding of the code.